kuksa_databroker/databroker/src/broker.rs

```diff
@@ -556,6 +558,18 @@ impl Entry {
         }
     }
 
+    fn validate_timestamp(&self, timestamp: &SystemTime) -> Result<(), UpdateError> {
+        if self.datapoint.ts > *timestamp {
+            return Err(UpdateError::TimestampTooOld);
+        }
+        if let Some(target) = &self.actuator_target {
```

Comment:
What's the use case for providing a timestamp when setting an actuator target in the first place?

My opinion is that setting an actuator is akin to calling a function. If it makes sense for clients to be able to read the timestamp of when this happened, I'm pretty sure this should always be the responsibility of databroker to set it.

So I would suggest:
1. Remove the validation of actuator target timestamp.
2. Remove the possibility of setting actuator target timestamps in the API.

Reply:
I would remove it, unless someone can come up with a reason why it would make sense for a client to provide a timestamp.